### ISSUE TITLE
characters on the map not doing anything in the simulation.

### ISSUE THREAD

**Issue by bowmanonjupiter2** [joonspk-research/generative_agents#108]
all 3 characters in test simulation , like base_the_ville_isabella_maria_klaus , no movement during the simulation. console log normal.

**Comment by thulanxc** [comment]
Because they are sleeping. Try simulating 2500 steps or more. IR will wake up at 6:00, and other two agents wake up later.

**Comment by bowmanonjupiter2** [comment]
indeed, thanks fellow. trying to make the time pass by faster , but seems update the sec_per_step in the meta.json doesn't work.

**Comment by wy12yw** [comment]
I have a k value error during the simulation at 6:59 am:UnboundLocalError: local variable 'k' referenced before assignment

**Comment by wy12yw** [comment]
I used a local open-source model to run the  generative town and had this issue: the agent kept sleeping from February 13th to February 14th.
